### DISCUSSION TITLE
where to change fu,mic in PK Sim

### DISCUSSION THREAD

**Discussion by Slow-Walker66** [Open-Systems-Pharmacology/Forum#1309] [Questions & Problems] [answered]
Hello, all

I wonder if there are some places to change the fu,mic in PK Sim?

Because when I modeling the DDI, I find that I have to change the Ki from 6 to 0.01 to match the clinical change, it's too exaggerated.

So I want to check the fu,mic value in PK Sim, however, I cannot find this parameter in PK Sim.

Or do you have any suggestions for when predicted value don't match the observed value in Drug drug interaction modeling.

Thanks you in advance!

**Comment by SRuedesh** [comment, marked as answer]
Hi @1998225, 

To my knowledge, there is no way to enter a fu,mic value directly in PK-Sim. However, you can adjust the respective parameter in PK-Sim (i.e. overwrite the value) to adjust for fu,mic. 
For instance, if a fu,mic value was determined/calculated for your microsomal assay, you would typically try to set the value of Ki to Ki * fu,mic. Make sure beforehand that the authors of your source have not already adjusted their reported values for fu,mic. If you have no information on microsomal binding in your assay, you could try approximation methods for fu,mic; for instance, have a look at this article by [Austin et al.](https://dmd.aspetjournals.org/content/30/12/1497)

Hope this helps,
Simeon

**Reply by Slow-Walker66** [reply]
Thanks, it helps a lot!
